x-pack/plugins/secops/public/components/formatted_duration/tooltip/index.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiToolTip } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { getHumanizedDuration } from '../helpers';
+
+const P = styled.p`
+  margin-bottom: 5px;
+`;
+
+export const FormattedDurationTooltipContent = pure<{
+  maybeDurationNanoseconds: string | number | object | undefined | null;
```

Comment:
I don't know if you want to mix styles or not, but just mentioning in other areas you had optional so this might be:

```ts
 maybeDurationNanoseconds?: string | number | object | null;
```

Just pointing it out in case that's not what you want.